docs/release-notes/index.rst

```diff
@@ -266,7 +266,18 @@ development keeps pace with the OS's new releases and support lifecycle.>
   Fixed bugs and issues
   ---------------------
 
-  The following issues have been resolved in Starcraft 2.0:
+  The following issues have been resolved in Starcraft 8.2.
```

Comment:
One more nit - will you make these 2.0, 2.0.0, and 2.0.1?  

The reason I ask is that I do a find-and-replace on `Starcraft` and `2.0` when I first copy the template, so this makes it easier to use the template.

Reply:
That makes more sense: [d04f44e](https://github.com/canonical/starbase/pull/375/commits/d04f44efcfae8af87637a341292b1cff4e2a5f36)